iphone/Classes/TiUIiPhoneProxy.m

```diff
@@ -226,10 +226,12 @@ -(NSString*)apiName
 
 BEGIN_UI_THREAD_PROTECTED_VALUE(statusBarHidden,NSNumber)
 result = [NSNumber numberWithBool:[[UIApplication sharedApplication] isStatusBarHidden]];
+DEPRECATED_REPLACED_REMOVED(@"UI.iPhone.statusBarHidden",@"3.1.3", @"6.0.0", @"UI.Window.fullscreen");
 END_UI_THREAD_PROTECTED_VALUE(statusBarHidden)
 
 BEGIN_UI_THREAD_PROTECTED_VALUE(statusBarStyle,NSNumber)
 result = [NSNumber numberWithInt:[[UIApplication sharedApplication] statusBarStyle]];
+DEPRECATED_REPLACED_REMOVED(@"UI.iPhone.statusBarStyle",@"3.1.3", @"6.0.0", @"UI.Window.statusBarStyle");
```

Comment:
I think this line would be printed everytime, something on the .iPhone. namespace is requested, because its not part of any method.